internal/pkg/template/workload.go

```diff
@@ -117,6 +118,13 @@ type Volume struct {
 	IAM           *string // ENABLED or DISABLED
 }
 
+// CopilotVolumeCreationInfo holds information about how to create Copilot-managed access points
+type CopilotVolumeCreationInfo struct {
```

Comment:
What do you think of 
```suggestion
type ManagedVolumeCreationInfo struct {
```